src/client/datascience/serviceRegistry.ts

```diff
@@ -231,6 +232,9 @@ export function registerTypes(serviceManager: IServiceManager, inNotebookApiExpe
     serviceManager.addSingleton<IExtensionSingleActivationService>(IExtensionSingleActivationService, NotebookUsageTracker);
     serviceManager.addSingleton<IExtensionSingleActivationService>(IExtensionSingleActivationService, MigrateJupyterInterpreterStateService);
     serviceManager.addSingleton<IExtensionSingleActivationService>(IExtensionSingleActivationService, VariableViewActivationService);
+    if (isDevMode) {
+        serviceManager.addSingleton<IExtensionSingleActivationService>(IExtensionSingleActivationService, LogReplayService);

```

Comment:
You'll notice it won't even be a part of any release bits.

Reply:
Not following you, do you mean to say this will not be bundled into the extension?
I thought it would

Reply:
Doesn't matter, dev only.